src/interfaces/article.d.ts

```diff
@@ -1,17 +1,21 @@
 import { EArticleType } from '@/constants/article';
 import { IGame } from './game';
 import { IUser } from './user';
+import { ITagSlim } from './tag';
+import { ICommentSlim } from './comment';
 
 export interface IArticle {
   id: number;
   game?: IGame;
   user?: IUser;
```

Comment:
```suggestion
  user: IUser;
```
这是之前加的测试文件  那时候还没有user, 现在article里面已经必有user了  可以把这个 问号去掉,  IUser可以换成自己想要的